### PR TITLE
feat: update scoring indicator

### PR DESCRIPTION
**Pull request by bartosjiri**
Update the scoring indicator to be more ✨ _fancy_ ✨

### CODE REVIEW

File: src/Explore/ScoreIndicator.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  return (
+    <Box
+      sx={{
+        position: 'relative',
+        width: 96,
+        background: 'linear-gradient(to right, #bb4677, #82b8e4)',
+        borderRadius: 8,
+        marginLeft: 1,
+        overflow: 'hidden',
```
Comment (axelboc) [inline src/Explore/ScoreIndicator.js:24]: If you put `z-index: 0` here, you shouldn't need to increase the `z-index` on the navigation bar.

File: src/Explore/ScoreIndicator.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      }}
+      onMouseEnter={handleHoverStart}
+      onTouchStart={handleHoverStart}
+      onMouseLeave={handleHoverEnd}
+      onTouchEnd={handleHoverEnd}
```
Comment (axelboc) [inline src/Explore/ScoreIndicator.js:29]: might be fine to just use `onPointerEnter` and `onPointerLeave`, I reckon - https://caniuse.com/pointer-events

File: src/Explore/ScoreIndicator.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    >
+      <Box
+        sx={{
+          position: 'relative',
+          width: '100%',
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
```
Comment (axelboc) [inline src/Explore/ScoreIndicator.js:38]: `alignItems: center` should work as well?

File: src/Explore/ScoreIndicator.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+
+import { Box } from '../Primitives'
+
+function ScoreIndicator({ score }) {
+  const [text, setText] = useState(score.toFixed(3))
```
Comment (axelboc) [inline src/Explore/ScoreIndicator.js:6]: I would maybe store a boolean, like `const [helpVisible, toggleHelp] = useToggle(false);`, and then just `toggleHelp()` in the event handlers. The help text and score can then be moved to the JSX, in a ternary condition.

**Review by axelboc — APPROVED** [review]
Very nice looking indeed! Thanks for the help :blush: 

Screenshots for reference:

image

File: src/Explore/ScoreIndicator.js (hunk 3 above)
Comment (bartosjiri) [reply src/Explore/ScoreIndicator.js:38]: Do you mean simply removing the `flexDirection`?

Comment (axelboc) [reply src/Explore/ScoreIndicator.js:38]: Actually, you may need to keep `justifyContent: 'center'` and remove `textAlign: center`. Oh well, not a big deal, it's fine either way.

Comment (bartosjiri) [reply src/Explore/ScoreIndicator.js:38]: Seems to be resulting to the following, if I understand you correctly.

image

Comment (axelboc) [reply src/Explore/ScoreIndicator.js:38]: Yeah, sorry I should have tested and given you a clearer code change suggestion -- don't worry about it, `flexDirection` is fine anyway.

**Pull request merged**